output_template/python/usb_camera_demo.py

```diff
@@ -79,6 +80,33 @@ def add_class_label(canvas,
     cv2.putText(canvas, text, dl_corner, font, font_scale, font_color, line_type)
 
 
+def label_to_color_image(results):
+    """Adds color defined by the dataset colormap to the label.
+
+    Args:
+        results: A 2D array with integer type, storing the segmentation label.
+
+    Returns:
+        result: A 2D array with floating type. The element of the array
+            is the color indexed by the corresponding element in the input label
+            to the PASCAL color map.
+
+    Raises:
+        ValueError: If label is not of rank 2 or its value is larger than color
+            map maximum entry.
+    """
+    if results.ndim != 4:
+        raise ValueError('Expect 4-D input results (1, height, width, classes).')
+
+    colormap = np.array(COLOR_MAP, dtype=np.uint8)
```

Comment:
@nlpng I want to use `get_color_map`. not `COLOR_MAP`
`get_color_map` support atbitrary length classes.

Reply:
@iizukak 
No problem. I will change it with next or following commits.

Reply:
OK Thanks!